acme/acme/client.py

```diff
@@ -609,6 +609,10 @@ def _send_request(self, method, url, *args, **kwargs):
                       method, url, args, kwargs)
         kwargs['verify'] = self.verify_ssl
         kwargs.setdefault('headers', {})
+        """
+          PULL REQUEST: fix bug where newline creeps in.
```

Comment:
You shouldn't put this text into the code; it isn't a thing that people want there in the future. Github will let you describe what you're doing in the pull request itself.